src/main/java/io/quarkus/backports/BackportsResource.java

```diff
@@ -62,6 +63,10 @@ public TemplateInstance backports(@NotNull(message = "Invalid Milestone")  @Rest
     public String markAsBackported(@NotNull(message = "Invalid Milestone") @RestPath Milestone milestone,
                                    @NotNull(message = "Invalid Pull Request") @RestPath PullRequest pullRequest) throws IOException {
         gitHub.markPullRequestAsBackported(pullRequest, milestone);
+        Log.info("    Backported PR to " + milestone.title + ": " + pullRequest.url);
+        pullRequest.commits.forEach(commit -> {
+            Log.info("    Backported commit to " + milestone.title + ": URL=" +  commit.url + ", message=" + commit.message);
```

Comment:
Milestone info is in line with PR details, I wouldn't repeat that
```suggestion
            Log.info("    Backported commit: URL=" +  commit.url + ", message=" + commit.message);
```